### PR TITLE
rotate etcd logs

### PR DESCRIPTION
**Pull request by mrandi**
fixes https://github.com/zalando-stups/taupage/issues/354

### CODE REVIEW

**Comment by mrandi** [comment]
👍

**Comment by ehartung** [comment]
:+1:

File: runtime/etc/logrotate.d/etcd [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/var/log/etcd.log
+{
+        rotate 1
```
Comment (szuecs) [inline runtime/etc/logrotate.d/etcd]: Why you only let 1 file and rotate every hour?

**Comment by szuecs** [comment]
:-1:

File: runtime/etc/logrotate.d/etcd (hunk 1 above)
Comment (mrandi) [inline runtime/etc/logrotate.d/etcd]: @szuecs What is the problem with this configuration? What would be your suggestion?
I always need the latest log on the machine, for older entries I can take a look also in other places.

Comment (szuecs) [inline runtime/etc/logrotate.d/etcd]: @mrandi Think about other people which may rely on having these logs.
My suggestion is at least have files for one week.

**Comment by mrandi** [comment]
👍

**Comment by szuecs** [comment]
:+1:

**Pull request merged**